stdlib/2and3/difflib.pyi

```diff
@@ -3,11 +3,17 @@
 import sys
 from typing import (
     TypeVar, Callable, Iterable, Iterator, List, NamedTuple, Sequence, Tuple,
-    Generic, Optional, Text, Union
+    Generic, Optional, Text, Union, AnyStr
 )
 
 _T = TypeVar('_T')
-_JunkCallback = Union[Callable[[Text], bool], Callable[[str], bool]]
+
+if sys.version_info >= (3,):
+    _StrType = Text
+    _JunkCallback = Callable[[str], bool]
```

Comment:
Why does this need to be conditional? Doesn't it resolve to the same thing in Python 3 anyway?